visma/gui/cli.py

```diff
@@ -12,6 +12,9 @@
 
 
 def commandExec(command):
+    if command.find('(') == -1:
```

Comment:
I think this check won't be enough for finding invalid(parenthesis) inputs. For example `simplify(` is an invalid input.

Reply:
i edited it, i think i resolved the issue this way.

Reply:
by checking if he inserted an open Parenthesis & closed Parenthesis & also the open must be before the closed one.